### PR TITLE
[parallel-hashmap] Update version to 1.3.11 

### PR DESCRIPTION
**Pull request by greg7mdp**
Specify library name and version:  **parallel-hashmap/1.3/11**

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
@greg7mdp parallel-hashmap recipe is already compatible with conan v2, I don't understand title of the PR.

File: recipes/parallel-hashmap/all/conanfile.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 class ParallelHashmapConan(ConanFile):
     name = "parallel-hashmap"
+    version = "1.3.11"
```
Comment (SpaceIm) [inline recipes/parallel-hashmap/all/conanfile.py:13]:
```suggestion
```

**Comment by greg7mdp** [comment]
> @greg7mdp parallel-hashmap recipe is already compatible with conan v2, I don't understand title of the PR.

My mistake, I changed the title.

File: recipes/parallel-hashmap/all/conanfile.py (hunk 1 above)
Comment (greg7mdp) [reply recipes/parallel-hashmap/all/conanfile.py]: I removed it, but when I run `conan create .` locally using conan v2, it fails if the version is not present.

**Pull request merged**